src/rust_tls.rs

```diff
@@ -42,38 +42,49 @@ mod cert {
     use std::io::ErrorKind;
     use std::path::Path;
 
-    use async_rustls::rustls::Certificate;
-    use async_rustls::rustls::PrivateKey;
-    use async_rustls::rustls::RootCertStore;
+    use futures_rustls::rustls::pki_types::CertificateDer;
+    use futures_rustls::rustls::pki_types::PrivateKeyDer;
+    use futures_rustls::rustls::RootCertStore;
     use rustls_pemfile::certs;
-    use rustls_pemfile::rsa_private_keys;
+    use rustls_pemfile::pkcs8_private_keys;
 
-    pub fn load_certs<P: AsRef<Path>>(path: P) -> Result<Vec<Certificate>, IoError> {
+    pub fn load_certs<P: AsRef<Path>>(path: P) -> Result<Vec<CertificateDer<'static>>, IoError> {
```

Comment:
since we are bump up major version, let's migrate this to anyhow

Reply:
done, looks a lot cleaner with anyhow